frontend/src/scenes/billing/v2/test/PlanTable.tsx

```diff
@@ -0,0 +1,478 @@
+                        value: true,
+                    },
+                    {
+                        name: 'Dashboards',
+                        value: true,
+                    },
+                    {
+                        name: 'Group analytics',
+                        value: true,
+                    },
+                ],
+            },
+            sessionRecording: {
+                name: 'Session Recording',
+                value: true,
+                note: 'Unlimited',
+            },
+            projects: {
+                name: 'Projects',
+                value: true,
+                note: 'Unlimited',
+            },
+            experiments: {
+                name: 'A/B testing',
+                value: true,
+            },
+            multivariateFFs: {
+                name: 'Multivariate feature flags',
+                value: true,
+            },
+            prioritySupport: {
+                name: 'Priority support',
+                value: true,
+                note: 'via Slack',
+            },
+            advancedPermissions: {
+                name: 'Advanced permissions',
+                value: true,
+            },
+            sso: {
+                name: 'SAML SSO',
+                value: true,
+            },
+        },
+        products: [
+            {
+                name: 'Product analytics',
+                tiers: [
+                    {
+                        description: 'First 1 million events/mo',
+                        price: 'Included',
+                    },
+                    {
+                        description: '1-2 million',
+                        price: '$0.0005625',
+                    },
+                ],
+            },
+            {
+                name: 'Session recording',
+                tiers: [
+                    {
+                        description: 'First 15,000 recordings/mo',
+                        price: 'Included',
+                    },
+                    {
+                        description: '15,000-50,000',
+                        price: '$0.00625',
+                    },
+                ],
+            },
+        ],
+    },
+]
+
+export function PlanTable({ redirectPath }: { redirectPath: string }): JSX.Element {
+    const { billing } = useValues(billingTestLogic)
+    const { reportBillingUpgradeClicked } = useActions(eventUsageLogic)
+
+    const upgradeButtons = billingPlans.map((plan) => (
+        <td key={`${plan.name}-cta`}>
+            <LemonButton
+                to={`${plan.signupLink}&redirect_path=${redirectPath}`}
+                type={plan.name === 'PostHog Cloud Lite' ? 'secondary' : 'primary'}
+                fullWidth
+                center
+                disableClientSideRouting
+                disabled={plan.name === 'PostHog Cloud Lite' && !billing?.billing_period}
+                onClick={() => {
+                    if (plan.name != 'PostHog Cloud Lite') {
```

Comment:
The "report billing billing upgraded click" should only be reported if clicking on non-`Lite` plans

Reply:
I did think about this... I kinda wanted to see if people clicked the button for the free plan for some reason, and in the future it will be used to downgrade so I figured it was easier to just leave it there instead of having to remember to remove the conditional.

Reply:
I would still not track this within this event, and just check that with some autocapture magic, so we don't have messy data